func/code.func

```diff
@@ -28,34 +28,36 @@ cell remove_old_orders(cell orders, int o_seqno) {
   return orders;
 }
 
-_ pack_state(int o_seqno, int n_wins, int in_amount, int out_amount, int hash, slice owner1, slice owner2, slice echo, cell orders) inline {
+_ pack_state(int o_seqno, int n_wins, int in_amount, int out_amount, int hash, slice owner, slice ecvrf, slice player, int random_locked, int player_coins, cell orders) inline {
   return begin_cell()             ;; FIRST CELL
     .store_uint(o_seqno, 64)      ;; 64 bit  | order seqno
     .store_uint(n_wins, 64)       ;; 64 bit  | number of wins
     .store_uint(in_amount, 256)   ;; 256 bit | ingoing amount
     .store_uint(out_amount, 256)  ;; 256 bit | outgoing amount
     .store_uint(hash, 256)        ;; 256 bit | seed salt
+    .store_uint(random_locked, 1) ;; 1 bit  | bool random_locked
     .store_dict(orders)           ;; bit+ref | last orders dict
     .store_ref(
       begin_cell()                ;; SECOND CELL:
-        .store_slice(owner1)      ;; 267 bit | owner 1, @sergei_ton
-        .store_slice(owner2)      ;; 267 bit | owner 2, @pyAndr3w
-        .store_slice(echo)        ;; 267 bit | crossworkchain echo contract
+        .store_slice(owner)      ;; 267 bit | owner 
+        .store_slice(ecvrf)        ;; 267 bit | ecvrf coordinator contract
+        .store_slice(player)      ;; 267 bit | current player
+        .store_coins(player_coins)
         .end_cell())
     .end_cell();
 }
 
 _ unpack_state() inline {
   var ds = get_data().begin_parse();
-  slice stats = ds~load_bits(896);
+  slice stats = ds~load_bits(896 + 64 + 32);
```

Comment:
The problem raising exit code 9 is here. There should be 897 bits loaded.

Reply:
ah! Yes should be 897 bits, let me try again with the changes